src/parser/priest/holy/modules/core/AbilityTracker.tsx

```diff
@@ -0,0 +1,7 @@
+import CoreAbilityTracker from 'parser/shared/modules/AbilityTracker';
+
+class AbilityTracker extends CoreAbilityTracker {
```

Comment:
You can just use CoreAbilityTracker in your CombatLogParser if you aren't extending or changing any functionality

(this might be a lie but im 87% sure)

Reply:
I'm planning on adding some stuff to the ability tracker, so I figured I'd just add the base module in this PR. I can remove it if you think I should for now.